src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String fileName) {
+        try {
+            Files.write(Paths.get(fileName), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write file" + fileName, e);
```

Comment:
The exception message should contain a space before the file name for readability. It should be "Can't write to file " + fileName.